src/wp-includes/media.php

```diff
@@ -1137,6 +1137,16 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 			}
 		}
 
+		// Adds 'auto' to the sizes attribute if applicable.
+		if (
+			isset( $attr['loading'] ) &&
+			'lazy' === $attr['loading'] &&
+			isset( $attr['sizes'] ) &&
```

Comment:
Minor enhancement: the issets can be combined into one:

```php
isset( $attr['loading'], $attr['sizes'] )
```

Reply:
Previously when i suggest this changes i got feedback from committer.

> Thanks for the suggestion! While technically true, I don't think we use that pattern with multiple variables anywhere else in core, and the current syntax seems a bit more readable to me.

Can we leave it for now if needed we can do it later.

Reply:
I personally find it more intuitive to read as it currently is too. I think combining such statements is only a good idea if that's the only condition in a clause.